src/System Application/App/AppSource Gallery/src/AppSourceProductManager.Codeunit.al

```diff
@@ -96,12 +96,16 @@ codeunit 2515 "AppSource Product Manager"
     /// <returns>True if the product can be installed, otherwise false</returns>
     internal procedure CanInstallProductWithPlans(Plans: JsonArray): Boolean
     var
+        AzureADUserManagement: Codeunit "Azure AD User Management";
         PlanToken: JsonToken;
         PlanObject: JsonObject;
         PricingTypesToken: JsonToken;
         PricingTypes: JsonArray;
         PricingType: JsonToken;
     begin
+        if (AzureADUserManagement.IsUserDelegated(Database.UserSecurityId())) then
```

Comment:
Not really a bug - but in AL we would not encapsulate a single condition in appetencies.  

```suggestion
        if AzureADUserManagement.IsUserDelegated(Database.UserSecurityId()) then
```

Reply:
I wonder why this isn't a CodeCop rule - I actually thought it was.